tutorials/notebooks/example_keras_imagenet.ipynb

```diff
@@ -1,482 +1,515 @@
+   },
+   "outputs": [],
+   "source": [
+    "TEST_DATASET_FOLDER = '/content/imagenet_val'\n",
+    "def get_validation_dataset():\n",
+    "    dataset = tf.keras.utils.image_dataset_from_directory(\n",
+    "        directory=TEST_DATASET_FOLDER,\n",
+    "        batch_size=BATCH_SIZE,\n",
+    "        image_size=[224, 224],\n",
+    "        shuffle=False,\n",
+    "        crop_to_aspect_ratio=True,\n",
+    "        interpolation='bilinear')\n",
+    "    dataset = dataset.map(lambda x, y: (imagenet_preprocess_input(x, y)))\n",
+    "    return dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fd661b39-e033-4efc-a916-f97a1642cb36",
+   "metadata": {
+    "id": "fd661b39-e033-4efc-a916-f97a1642cb36"
+   },
+   "outputs": [],
+   "source": [
+    "evaluation_dataset = get_validation_dataset()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9889d217-90a6-4615-8569-38dc9cdd5999",
+   "metadata": {
+    "id": "9889d217-90a6-4615-8569-38dc9cdd5999"
+   },
+   "source": [
+    "Let's start with the floating-point model evaluation.\n",
+    "\n",
+    "We need to compile the model before evaluation and set the loss and the evaluation metric:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1d3a0ae9-beaa-4af8-8481-49d4917c2209",
+   "metadata": {
+    "id": "1d3a0ae9-beaa-4af8-8481-49d4917c2209"
+   },
+   "outputs": [],
+   "source": [
+    "float_model.compile(loss=keras.losses.SparseCategoricalCrossentropy(), metrics=[\"accuracy\"])\n",
+    "results = float_model.evaluate(evaluation_dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ead4a6f3-86a0-4e6c-8229-a2ff514f7b8c",
+   "metadata": {
+    "id": "ead4a6f3-86a0-4e6c-8229-a2ff514f7b8c"
+   },
+   "source": [
+    "Finally, let's evaluate the quantized model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1bc377ee-39b4-4ced-95db-f7d51ab60848",
+   "metadata": {
+    "id": "1bc377ee-39b4-4ced-95db-f7d51ab60848"
+   },
+   "outputs": [],
+   "source": [
+    "quantized_model.compile(loss=keras.losses.SparseCategoricalCrossentropy(), metrics=[\"accuracy\"])\n",
+    "results = quantized_model.evaluate(evaluation_dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebfbb4de-5b6e-4732-83d3-a21e96cdd866",
+   "metadata": {
+    "id": "ebfbb4de-5b6e-4732-83d3-a21e96cdd866"
+   },
+   "source": [
+    "You can see that we got a very small degradation with a compression rate of x4 !"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Now, we can export the model to Keras and TFLite:"
+   ],
+   "metadata": {
+    "id": "6YjIdiRRjgkL"
+   },
+   "id": "6YjIdiRRjgkL"
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "import tempfile\n",
+    "_, tflite_file_path = tempfile.mkstemp('.tflite') # Path of exported model\n",
+    "mct.exporter.keras_export_model(model=quantized_model, save_model_path=tflite_file_path,\n",
```

Comment:
1. better to just save it locally: replace "tflite_file_path" with the string "qmodel.tflite". Same for the keras export. Then delete the tempfile import
2. this code will fails on the latest MCT release, so better to install mct-nightly